scalameta/scalameta/shared/src/test/scala/scala/meta/tests/prettyprinters/PublicSuite.scala

```diff
@@ -133,10 +139,10 @@ class PublicSuite extends FunSuite {
   test("scala.meta.inputs.Input.File.toString") {
     import java.io._
     import java.nio.charset.Charset
-    val file = new File("hello.scala")
-    val input1 = Input.File(file, Charset.forName("cp1251"))
-    val input2 = Input.File(file, Charset.forName("UTF-8"))
-    assert(input1.toString == """Input.File(new File("hello.scala"), Charset.forName("windows-1251"))""")
+    val path = AbsolutePath("hello.scala")
```

Comment:
From our initial discussion, I got the impression that `AbsolutePath` is only going to take absolute paths in its constructor. This line challenges this impression. Could you elaborate?